lib/beaker/tasks/quick_start.rb

```diff
@@ -9,17 +9,17 @@
 
   desc 'Generate Default Beaker Host Config File, valid options are: vmpooler or vagrant.'
   task :gen_hosts, [:hypervisor] do |t, args|
-    hosts_file = "acceptance/config/default_#{args[:hypervisor]}_hosts.yaml"
+    hosts_file = ".beaker/acceptance/config/default_#{args[:hypervisor]}_hosts.yaml"
     if args[:hypervisor] == 'vagrant'
       cli = VAGRANT
     elsif args[:hypervisor] == 'vmpooler'
       cli = VMPOOLER
     else
       puts "No hypervisor provided, defaulting to vagrant."
-      hosts_file = "acceptance/config/default_vagrant_hosts.yaml"
+      hosts_file = ".beaker/acceptance/config/default_vagrant_hosts.yaml"
       cli = VAGRANT
     end
-    FileUtils.mkdir_p('acceptance/config') # -p ignores when dir already exists
+    FileUtils.mkdir_p(".beaker/acceptance/config") # -p ignores when dir already exists
```

Comment:
@ferglor since we're repeating the `.beaker/acceptance/config` path 3x here, it would be an improvement to pull this path out into a constant.